docs/data/joy/getting-started/templates/email/App.tsx

```diff
@@ -176,42 +103,34 @@ export default function EmailExample() {
           <Box
             sx={{
               p: 2,
-              mb: 1,
               display: 'flex',
               alignItems: 'center',
               justifyContent: 'space-between',
             }}
           >
-            <Typography level="title-sm" textColor="text.tertiary">
-              Unread
-            </Typography>
-            <IconButton size="sm" variant="plain" color="primary">
-              <KeyboardArrowDownRoundedIcon fontSize="small" color="primary" />
-            </IconButton>
-          </Box>
-          <Box sx={{ py: 10 }}>
-            <Typography
-              textColor="text.tertiary"
-              level="body-sm"
-              sx={{ textAlign: 'center' }}
+            <Box
+              sx={{
+                alignItems: 'center',
+                gap: 1,
+              }}
             >
-              You&apos;ve read all messages in your inbox.
-            </Typography>
-          </Box>
-          <Box
-            sx={{
-              p: 2,
-              display: 'flex',
-              alignItems: 'center',
-              justifyContent: 'space-between',
-            }}
-          >
-            <Typography level="title-sm" textColor="text.tertiary">
-              Everything else
-            </Typography>
-            <IconButton size="sm" variant="plain" color="primary">
-              <KeyboardArrowDownRoundedIcon fontSize="small" color="primary" />
-            </IconButton>
+              <Typography level="title-lg" textColor="text.secondary">
+                My inbox
+              </Typography>
+              <Typography level="title-sm" textColor="text.tertiary">
+                5 emails
+              </Typography>
+            </Box>
+            <Button
+              size="sm"
+              startDecorator={<CreateRoundedIcon />}
+              onClick={() => setOpen(true)}
```

Comment:
```suggestion
              onClick={() => setOpen(true)}
              sx={{ ml: 'auto' }}
```
This will push the button to the right.

Current bug:
<img width="534" alt="image" src="https://github.com/mui/material-ui/assets/18292247/973d5605-8b3e-4a7f-bf0f-8b59c22e8d3c">